src/components/BatchAllocations/BatchAllocationLogs/constants.js

```diff
@@ -0,0 +1,22 @@
+export const BATCH_ALLOCATION_LOG_FIELDS = {
+  jobName: 'jobName',
+  status: 'status',
+  recordsCount: 'recordsCount',
+  createdDate: 'createdDate',
+  updatedDate: 'updatedDate',
+  createdByUsername: 'createdByUsername',
+  jobNumber: 'jobNumber',
+};
+
+const select = 'select';
```

Comment:
I haven't found where it used and I don't think we have to have this const at all to be honest